example/lib/customs/pickers/directory_file_asset_picker.dart

```diff
@@ -584,7 +584,7 @@ class FileAssetPickerBuilder
   Widget assetsGridBuilder(BuildContext context) {
     appBarPreferredSize ??= appBar(context).preferredSize;
     int totalCount = provider.currentAssets.length;
-    if (specialItemPosition != SpecialItemPosition.none) {
+    if (specialItems.isNotEmpty) {
       totalCount += 1;
```

Comment:
Here we should add the items' count, correct?

Reply:
yes, done updated.